pkg/daemon/constants/constants.go

```diff
@@ -100,6 +100,9 @@ const (
 	// changes to registries.d will cause a crio reload
 	SigstoreRegistriesConfigDir = "/etc/containers/registries.d"
 
+	// changes to /etc/crio/policies will cause a crio reload
+	CrioPoliciesDir = "/etc/crio/policies"
```

Comment:
I think we have to wire MCD up to skip reboots for this directory. Did we?

Reply:
yes, the node drain policies is in https://github.com/openshift/machine-config-operator/pull/4587/files#diff-679de7a0ee9e32bf2764789dc26e565871129d7c2ff80a489ab9632c33ae3136